appveyor.yml

```diff
@@ -1,3 +1,24 @@
+# The config expects the following environment variables to be set:
+#  - "GN_CONFIG" Build type. One of {'debug', 'testing', 'release'}.
+#  - "GN_EXTRA_ARGS" Additional gn arguments for a build config,
+#      e.g. 'target_cpu="x86"' to build for a 32bit platform.
+#      https://gn.googlesource.com/gn/+/master/docs/reference.md#target_cpu
+#  - "ELECTRON_RELEASE" Define it to upload binaries on success.
+#  - "NPM_CONFIG_ARCH" E.g. 'x86'. Is used to build native Node.js modules.
```

Comment:
should this be set based on `TARGET_ARCH`? if not, mention in what cases it should differ from `TARGET_ARCH`.

Reply:
Both `NPM_CONFIG_ARCH` and `TARGET_ARCH` shouldn't exist at all.
Because none of them actually control the `target_cpu` arg passes to gn.

> in what cases it should differ from `TARGET_ARCH`.
Right now they all must have the same value.

Reply:
Great, please add this context to the comment!

Reply:
I kinda hate making more changes and waiting for CI jobs again, but your comments make sense, so ok... fine... =)